File: app/src/main/java/moe/tlaster/openween/core/model/status/RepostListModel.java
package moe.tlaster.openween.core.model.status;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import moe.tlaster.openween.core.model.BaseListModel;

/**
 * Created by Tlaster on 2016/9/2.
 */
public class RepostListModel extends BaseListModel {

    @SerializedName("reposts")
    public List<MessageModel> mReposts;

    public List<MessageModel> getReposts() {
        return mReposts;
    }

    public void setReposts(List<MessageModel> reposts) {
        mReposts = reposts;
    }
}

File: app/src/main/kotlin/moe/tlaster/openween/core/model/status/RepostListModel.kt
package moe.tlaster.openween.core.model.status

import com.google.gson.annotations.SerializedName

import moe.tlaster.openween.core.model.BaseListModel

/**
 * Created by Tlaster on 2016/9/2.
 */
class RepostListModel : BaseListModel() {

    @SerializedName("reposts")
    var reposts: List<MessageModel>? = null
}
